From 2f6a830958cd75e15da083bf52005ff0643ee002 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Bene=C5=A1?= <martin.benes@stratox.com>
Date: Sun, 8 Oct 2023 22:29:14 +0900
Subject: [PATCH] add udev rule for micmute (f20)

(cherry picked from commit fc48807d2937891305fc486022d0327fb2b10ae5)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index aa5dc5308b..e981ff4e14 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -221,6 +221,10 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGateway*:pn*:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPackard*Bell*:pn*:*
  KEYBOARD_KEY_86=wlan                                   # Fn+F3 or Fn+Q for communication key
 
+# Swift SF314-511
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnSwiftSF314-511:pvr*
+ KEYBOARD_KEY_8a=f20					# Fn+F12, microphone mute
+
 ###########################################################
 # Alienware
 ###########################################################
